example/include/Controller.h

```diff
@@ -0,0 +1,19 @@
+// SPDX-FileCopyrightText: Deutsches Elektronen-Synchrotron DESY, MSK, ChimeraTK Project <chimeratk-support@desy.de>
+// SPDX-License-Identifier: LGPL-3.0-or-later
+#pragma once
+
+#include <ChimeraTK/ApplicationCore/ApplicationCore.h>
+
+namespace ctk = ChimeraTK;
+
+//! [Class Definition]
+class Controller : public ctk::ApplicationModule {
+  using ctk::ApplicationModule::ApplicationModule;
+
+  ctk::ScalarPollInput<float> sp{this, "temperatureSetpoint", "degC", "Setpoint for the temperature controller"};
```

Comment:
Usually we would not use variable names as short as sp, rb and cur. We introduced this very short form in order to safe space on the presentation slides.
According to our coding style they should be more verbose because they are class wide, not local.